app/views/Mails/acceptrefuse/sendProposalRefused.scala.txt

```diff
@@ -3,36 +3,29 @@
 @if(ConferenceDescriptor.current().frLangEnabled) {
 Bonjour,
 
-Nous avons le regret de vous informer que votre proposition @proposal.title au format @Messages(proposal.talkType.id) n'a pas été acceptée.
+Nous avons le regret de vous informer que votre proposition @proposal.title (@Messages(proposal.talkType.id)) n'a pas été sélectionnée.
+Nous vous remercions pour le temps que vous avez consacré à préparer votre sujet.
 
-Nous vous remercions cependant pour le temps que vous avez consacré à préparer votre sujet.
-Si vous souhaitez vous inscrire, la conférence étant complète, nous avons mis en place un système d'inscription limité à une place maximum
-sur le site de la conférence Devoxx France :
+Si vous souhaitez vous inscrire, la conférence étant complète, nous avons mis en place un système d'inscription limité à une place maximum sur le site de la conférence Devoxx France :
 
 https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email
 
-Si par ailleurs vous avez eu la chance d'avoir un autre sujet sélectionné et que ce sujet vous donne un badge speaker :
-ne partagez pas ce lien svp.
+Si par ailleurs vous avez eu la chance d'avoir un autre sujet sélectionné et que ce sujet vous donne un badge speaker : ne partagez pas ce lien svp.
 Il est lié à votre présentation et il ne permet pas l'inscription d'autres personnes.
 Si vous avez besoin d'acheter plus d'une place, merci de nous contacter par email: info@@devoxx.fr
 
-La sélection cette année a encore été difficile avec près de 800 propositions pour 200 sessions.
-
 Merci pour votre proposition,
-L'équipe de sélection de la conférence @Messages("longYearlyName")
+L'équipe @Messages("longYearlyName")
 
 ----------------------------------------------------------------------------------------------------------------
 }
 Hi,
-We're sorry to inform you that your proposal for @Messages("longYearlyName") was refused, keep in mind that
-we received close to 800 proposals and the available slots are very limited.
-
-Proposition title : @proposal.title, @Messages(proposal.talkType.id).
 
+We're sorry to inform you that your proposal @proposal.title (@Messages(proposal.talkType.id)) for @Messages("longYearlyName") was refused.
 
 We would like to thank you sincerely for your proposal and shown interest.
-Hopefully we can still welcome you at @Messages("longYearlyName"), registration details
-You can still register using a dedicated page for you here :
+Hopefully we can still welcome you at @Messages("longYearlyName"), we set up a special service for refused proposals, so that you can register :
```

Comment:
```suggestion
Hopefully we can still welcome you at @Messages("longYearlyName"), we have set up a special service for rejected proposals, so that you can register :
```